application/src/main/data/json/edge/instructions/upgrade/upgrade_db.md

```diff
@@ -0,0 +1,8 @@
+**NOTE**: Package installer may ask you to merge your tb-edge configuration. It is preferred to use **merge option** to make sure that all your previous parameters will not be overwritten.
+
+Execute regular upgrade script:
+
+```bash
+sudo /usr/share/tb-edge/bin/install/upgrade.sh --fromVersion=${CURRENT_TB_EDGE_VERSION}
```

Comment:
I believe current is not corrent label. 
Here is more suitable FROM:
${FROM_TB_EDGE_VERSION}